### ISSUE TITLE
Add Clojure support

### ISSUE THREAD

**Issue by mhalano** [ubuntu/ubuntu-make#105]
Clojure is a alternative for Java and there is a IDE for them called Nightcode: https://sekao.net/nightcode/

**Comment by wxl** [comment]
tl;dr re: Clojure +1

Clojure is an alternative for Java? Ehhhh, I don't know about that. It's a general purpose Lisp dialect, with a heavy emphasis on functional programming, targetted at the JVM so as to essentially build on the shoulders of giants. It can be used to write Java and it does compile to JVM bytecode, but I wouldn't call it an alternative to Java. I mean what Java programmer could put up with all those parens, let alone prefix notation, not to mention the lack of ridiculous verbosity?! I digress…

Nightcode looks really nice and is public domain! Sure beats setting up Emacs and all of the plugins needed to make it useful (though terminal folks will certainly prefer it). There's also [Light Table][], which is a little bit more of ambitious project. Between the two, I would probably suggest it, since it has less Java (ew), is extensible through plugins, and really aims to be more than Nightcode is. It's more complex, so maybe Nightcode is nice from that perspective, but I still say Light Table.

[Light Table]: http://lighttable.com/
